### ISSUE TITLE
Allow curl module to be used as http module

### ISSUE THREAD

**Issue by felipefoz** [buildinspace/peru#217]
As mentioned and briefly discussed in #195 by @oconnor663 and @colindean, the curl module today in peru is not actually using curl, which is confusing and can lead to wrong conclusions.

The proposal here is to create an alias for the curl module to be also recognized as http.

And then in future versions we keep only http as the module using the requests library.

**Comment by oconnor663** [comment]
Could we be concrete about what confusions result from the current name. For example, what specific things does Curl do with e.g. your `~/.netrc` file, where peru might surprise people by not doing that thing? Is there anything besides `~/.netrc`?

**Comment by felipefoz** [comment]
Yes, I believe software should clearly inform its intent. In this case, the curl module make us think that curl is used for downloading the packages, as the git module uses git for downloading git repositories.
Wouldn't it be a surprise for you that the git module actually was using other software than git?

That happened to me when I discovered that curl was not actually curl, while I haven't explored yet all that curl can do differently besides .netrc, as you mentioned.

The point is: the name is not suitable and even http might not be either, but we can be more clear on that.
